File: src/main/java/com/project/ecoWater/filling/domain/TankFilling.java
package com.project.ecoWater.filling.domain;


import com.project.ecoWater.tank.app.TankDTO;
import lombok.Builder;
import lombok.Data;

import java.math.BigDecimal;
import java.sql.Timestamp;

@Data
@Builder
public class TankFilling {
    private Long fillingId;
    private BigDecimal totalVolume;
    private Timestamp startedDate;
    private Timestamp finishedDate;
    private TankDTO tank;
}

File: frontend/Proyecto/Proyecto/app/src/main/java/com/example/proyecto/model/tankFilling/TankFilling.kt
package com.example.proyecto.model.tankFilling

import com.example.proyecto.model.tank.TankDTO
import java.sql.Timestamp

data class TankFilling (
    val tankFillingId: Long? = null,
    val tank: TankDTO? = null,
    val startedDate: Timestamp? = null,
    val finishedDate: Timestamp? = null,
    val totalVolume: Float? = null
)